src/Adapter/MSTest.TestAdapter/SourceGeneration/SourceGeneratedReflectionOperations.cs

```diff
@@ -68,7 +68,7 @@ public IEnumerable<PropertyInfo> GetDeclaredProperties(Type type)
     public Type[] GetDefinedTypes(Assembly assembly)
         => ReflectionDataProvider.Types;
 
-    public IEnumerable<MethodInfo> GetRuntimeMethods(Type type)
+    public MethodInfo[] GetRuntimeMethods(Type type)
         => ReflectionDataProvider.TypeMethods[type];
```

Comment:
The return type here was already an array, but making at strongly-typed allows callsites to not allocate an enumerator and be more efficient.